src/frontend/src/lib/components/ui/ExternalLink.svelte

```diff
@@ -14,7 +14,7 @@
 	{href}
 	rel="external noopener noreferrer"
 	target="_blank"
-	class="inline-flex items-center gap-2 no-underline"
+	class="inline-flex items-center gap-2 no-underline {$$restProps.class || ''}"
```

Comment:
Shouldn't we provide this in another PR? Seems unrelated.

Reply:
In my opinion not - This change is necessary to implement this modal. But let me know if you want a separate PR and I will make one.

Reply:
on a separate note, should there be `nonNullish` here or `??` coalescing? I thought the linter would spot the second one...

Reply:
I think this should have been a separate PR because we don't spread classes but use a property named `styleClass` when passing a class to a component. However, if you want to keep this in this PR, please adjust the code accordingly for consistency reasons.